runtests.sh

```diff
@@ -17,7 +17,7 @@ main() {
   git grep -El '^#!/.+\b(bash|sh)\b' -- './*' ':(exclude)*.tmpl' | xargs shellcheck
 
   echo -e '\n-----> Running terraform validate'
-  for d in $(git ls-files '*.tf' | xargs -n1 dirname | LC_ALL=C sort | uniq); do
+  for d in $(git ls-files '*.tf' | xargs -n1 dirname | LC_ALL=C sort | grep -v . | uniq); do
```

Comment:
won't `grep -v .` exclude all lines?  you probably want `grep -v '^\.$'`.

Reply:
I can never remember the default rules for when grep uses regexp. It may even vary by grep implementation. I changed this to `grep -E` just to be sure.

Reply:
grep always uses a regex by default; use `-f` for substring matching.